ntoskrnl/include/internal/icif.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+/*
```

Comment:
A comment explaining why this is separate and can't depend on other kernel headers would be good.

Reply:
The purpose here is to have information class interface stuff into own separate header as you originally asked me to do so it can be accessible across different sources (the testcase suite here for instance).

Reply:
Oh I understand -- I mean the code should have a comment.